transformers/events/auction_file/transformer.go

```diff
@@ -0,0 +1,62 @@
+package auction_file
+
+import (
+	"fmt"
+
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	"github.com/makerdao/vulcanizedb/pkg/core"
+	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
+)
+
+type Transformer struct{}
+
+func (t Transformer) ToModels(_ string, logs []core.EventLog, db *postgres.DB) ([]event.InsertionModel, error) {
+	var models []event.InsertionModel
+	for _, log := range logs {
+		err := shared.VerifyLog(log.Log, shared.ThreeTopicsRequired, shared.LogDataNotRequired)
```

Comment:
I forget how we determine how many topics are required, but should this be `FourTopicsRequired`? `Signature` [0], `msg.sender` [1], `arg1` [2], and `arg2` [3]?

Reply:
yes - nice catch! I think this might actually apply to some of the other `file` transformers (🍝) - so may add a separate commit to update them